Update to fix a minor typo on cosmos-db/serverless-computing-database

SudevG opened the pull request:
Line#135 should be referring to Azure "Blob" storage instead of "Blog" storage.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

SudevG committed:
Update to fix a minor typo.

Line#135 should be referring to Azure "Blob" storage instead of "Blog" storage.
